tutordistro/templates/distro/tasks/lms/init

```diff
@@ -0,0 +1,8 @@
+{% if DISTRO_DISABLE_MFE %}
+(./manage.py lms waffle_flag --list | grep course_home.course_home_use_legacy_frontend) || ./manage.py lms waffle_flag --create --everyone course_home.course_home_use_legacy_frontend
```

Comment:
Do we need this part `(./manage.py LMS waffle_flag --list | grep course_home.course_home_use_legacy_frontend)`? Why do we need that? Is it not enough with the other command that creates the flag?

Reply:
This part checks if flags are already created and not duplicated. I use  tutor-mfe  like inspiration for waffle flags creation https://github.com/overhangio/tutor-mfe/blob/master/tutormfe/templates/mfe/tasks/lms/init